pkgs/development/tools/java/jextract/default.nix

```diff
@@ -49,9 +49,14 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = [
     gradle
+    jdk20
     makeWrapper
   ];
 
+  buildInputs = [
+    llvmPackages.libclang
+  ];
```

Comment:
Just drop this part (and the jdk20 in nativeBuildInputs) as the hook does not work?

Reply:
Dropped.